sdk/appconfiguration/azure-appconfiguration/tests/perfstress_tests/get_set.py

```diff
@@ -0,0 +1,42 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure_devtools.perfstress_tests import PerfStressTest
+
+from azure.appconfiguration import ConfigurationSetting, AzureAppConfigurationClient as SyncAppConfigClient
+from azure.appconfiguration.aio import AzureAppConfigurationClient as AsyncAppConfigClient
+
+
+class GetSetTest(PerfStressTest):
+    service_client = None
+    async_service_client = None
+
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        connection_string = self.get_from_env("AZURE_APP_CONFIG_CONNECTION_STRING")
+        self.service_client = SyncAppConfigClient.from_connection_string(connection_string=connection_string)
+        self.async_service_client = AsyncAppConfigClient.from_connection_string(connection_string=connection_string)
+
+    async def close(self):
+        # await self.async_service_client.close()
+        await super().close()
+
+    def run_sync(self):
+        kv = ConfigurationSetting(
+            key="KEY",
+            value="VALUE",
+        )
+        self.service_client.set_configuration_setting(kv)
+        self.service_client.get_configuration_setting(key=kv.key)
```

Comment:
Was it decided cross-language to use both these operations in a single perf test?
Typically for the other SDKs we have tried to make the perf test as singular as possible.
For example - if we determine that Python is running this scenario slower than JS, is it because of the Get operation or the Set operation?
I would break this into two separate tests, each one may need some setup/cleanup:
- Get config setting, where each class has setup function that does the Set operation with a UUID for key, then a cleanup operation to remove that setting
- Set config setting, where each class has a unique UUID key value to set, which is then cleaned up after completion.

Reply:
No. This is my proposed scenario.

In app config, set and get almost do nothing other than call the API can translate the object.

The most (only) tricky part is to parse the sync-token.

So I want to try set (the service will generate a new sync-token) and then get (client will use the sync-token to get the new value).

This is open and your comments are more than welcome.